### ISSUE TITLE
Help Needed...ComboBox

### ISSUE THREAD

**Issue by kannanrs** [yiszza/Material.Components.Maui#57]
Hi..

How to add items to Combobox in Codebehind..

TIA

**Comment by ziomek64** [comment]
Bind items or items source I think in combobox to some collection in your code. Add items to this collection. Haven't used it but something like that perhaps. You have list of properties in docs https://mdc-maui.github.io/combo-box. 

If you by any chance don't know what binding is learn that first and how mvvm works

**Comment by kannanrs** [comment]
Will update soon

**Comment by Chupakabra0** [comment]
> Bind items or items source

Seems to be binding for `ComboBox` properties like `Items` and `ItemsSource` doesn't work at least at the actual preview version. I've tried to bind it with `List<string>`, `ObservableCollection<string>`. I've even tried to bind it with `List<Material.Components.Maui.MenuItem>`, used converters, etc. Nothing helped.

Tried to do it by editing the sample project from `develop` brunch. Same sad result, but sample project from `master` worked perfectly (but somehow it breaks Android build so still sad).

**Comment by yiszza** [comment]
Copy!

**Comment by kannanrs** [comment]
> > Bind items or items source
> 
> Seems to be binding for `ComboBox` properties like `Items` and `ItemsSource` doesn't work at least at the actual preview version. I've tried to bind it with `List<string>`, `ObservableCollection<string>`. I've even tried to bind it with `List<Material.Components.Maui.MenuItem>`, used converters, etc. Nothing helped.
> 
> Tried to do it by editing the sample project from `develop` brunch. Same sad result, but sample project from `master` worked perfectly (but somehow it breaks Android build so still sad).

yes.. did all that.. and still couldnt... can wait... why not...

**Comment by sthakkar0687** [comment]
I am able to add items to combobox from code behind file.  But You will Have to Make Observable Collection of Material.Components.Maui.MenuItem.  And you will have to Add to the Items Property in combobox.


For eg  there are 2 ways : 
1 ) Way to add single Item through foreach loop: 

foreach(var item in myCollection)
{
combobox1.Items.Add(new Material.Components.Maui.MenuItem { Text = item.PropertyName });
}


2) Way to add collection of Items by making Observable Collection.

Use Linq. 

var cmbItemsCollection = new ObservableCollection<Material.Components.Maui.MenuItem>(mycollection.Select(p=> new Material.Components.Maui.MenuItem() { Text = p.PropertyName }).ToList());

//Now add the observable collection in the Combobox items : 

combobox1.Items.AddRange<Material.Components.Maui.MenuItem>(cmbItemsCollection);


Adding items is done , but When there are many items, there is no scrollbar coming. 
If anybody finds a way to do that, Please share.

Thank you,
Sagar Thakkar

**Comment by kannanrs** [comment]
guess new update will sort out these things... guess @yiszza has time...

Regards

**Comment by kannanrs** [comment]
any updates please.....

**Comment by kannanrs** [comment]
Hi,

Need help.. Projects are not compiling. iOs and MacCatalyst  return error. Project building fails for now... for what has been running so well...


TIA
